test/acceptance/integration/cb_mdw/cb_actions_domain/cb_actions_domain_rol.feature

```diff
@@ -127,23 +127,6 @@ Feature: Context broker actions when the role is defined only in a domain
     | xml    |
     | json   |
 
-  Scenario Outline: subscribe-availability Standard operation
```

Comment:
Not sure if .feature files in this repos are still mantained, but I has been easy to remove the scenarios I have find (grep-wise) related with subscribe-availability

Reply:
NTC